### DISCUSSION TITLE
How to "loopdata through Google Sheets" from first row while "resume last workflow is enabled"?

### DISCUSSION THREAD

**Discussion by alex223ku** [AutomaApp/automa#1540] [Q&A]
first time, it works perfectly.
second time execute the workflow, it always repeat the same rows.

besides, what is the difference from "Reuse last workflow state" in setting

**Comment by Delusionnnn** [comment]
> first time, it works perfectly. second time execute the workflow, it always repeat the same rows.
> 
> besides, what is the difference from "Reuse last workflow state" in setting

Resume last workflow means it will resume looping through the data from where it left off the last time you ran the workflow.
